packages/zeebe-element-templates-json-schema/test/fixtures/placeholder-invalid-property.js

```diff
@@ -0,0 +1,72 @@
+export const template = {
```

Comment:
Question: We decided to only support it for C8?

Reply:
We could implement it for both platforms, but that means greater effort in bpmn-js-element-templates project.

Reply:
Let's have it in the core then.